tcpdf.php

```diff
@@ -7558,6 +7558,8 @@ public function Output($name='doc.pdf', $dest='I') {
 		}
 		$dest = strtoupper($dest);
 		if ($dest[0] != 'F') {
+			$name = explode("/", $name);
+			$name = $name[count($name) - 1];
```

Comment:
Why only make sure that the basename should contain proper characters? Doesn't this create a problem and potential bug if anything but the basename contains unsupported characters?